optaplanner-persistence/optaplanner-persistence-jaxb/pom.xml

```diff
@@ -51,23 +51,22 @@
     </dependency>
     <!-- XML and JSON -->
     <dependency>
-      <groupId>org.jboss.spec.javax.xml.bind</groupId>
-      <artifactId>jboss-jaxb-api_2.3_spec</artifactId>
+      <groupId>jakarta.xml.bind</groupId>
+      <artifactId>jakarta.xml.bind-api</artifactId>
     </dependency>
     <dependency>
       <groupId>org.glassfish.jaxb</groupId>
       <artifactId>jaxb-runtime</artifactId>
       <scope>test</scope>
-      <exclusions>
-        <exclusion>
-          <groupId>jakarta.xml.bind</groupId>
-          <artifactId>jakarta.xml.bind-api</artifactId>
-        </exclusion>
-      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>jakarta.json</groupId>
+      <artifactId>jakarta.json-api</artifactId>
+      <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.glassfish</groupId>
-      <artifactId>jakarta.json</artifactId>
+      <artifactId>javax.json</artifactId>
```

Comment:
This PR mostly changes `javax` artifacts to their `jakarta` alternatives. Is there a reason why this particular dependency moves in the opposite direction?
The `jakarta.json` seems to be more recent according to https://mvnrepository.com/artifact/org.glassfish/jakarta.json and https://mvnrepository.com/artifact/org.glassfish/javax.json.